extensions/3DTILES_implicit_tiling/0.0.0/schema/3DTILES_implicit_tiling.schema.json

```diff
@@ -4,38 +4,24 @@
     "type": "object",
     "description": "This extension enables 3D Tiles to support tilesets with implied subdivision schemes",
     "properties": {
-      "boundingVolume": {
-        "allOf": [{"$ref": "boundingVolume.schema.json"}],
-        "description": "The bounding volume that encloses the root tile. The bounding volume is subdivided at the midpoint for each level of subdivision according to the tiling scheme."
-      },
-      "refine": {
-        "type": "string",
-        "description": "Specifies if additive or replacement refinement is used when traversing the tileset. This property applies to all tiles in the tileset.",
-        "enum" : ["ADD", "REPLACE"]
-      },
-      "tilingScheme": {
+      "subdivisionScheme": {
         "type": "string",
-        "description": "A string describing the tiling scheme used within the tileset.",
+        "description": "A string describing the subdivision scheme used within the tileset.",
         "enum" : ["QUADTREE", "OCTREE"]
       },
-      "geometricError": {
-        "type": "number",
-        "description": "The error, in meters, introduced if the root is rendered and its children are not. The geometric error is halved for each level of subdivision. At runtime, the geometric error is used to compute screen space error (SSE), i.e., the error measured in pixels.",
-        "minimum" : 0
-      },
-      "subtreeDepth": {
+      "subtreeLevels": {
         "type": "integer",
-        "description": "The depth of each subtree.",
+        "description": "The number of distinct levels in each subtree. For example, a quadtree with `subtreeLevels = 2` will have subtrees with 5 nodes (one root and 4 children)",
         "minimum" : 1
       },
-      "maximumDepth": {
+      "maximumLevel": {
         "type": "integer",
         "description": "The maximum depth of the tree.",
         "minimum" : 0
       },
       "subtrees": {
         "allOf": [{"$ref": "templateUri.schema.json"}],
```

Comment:
not in this file, but in `templateUri.schema.json`: Update it to use `{x}` syntax.